website/docs/getting-started/setup/installation-guides/docker/README.md

```diff
@@ -5,172 +5,157 @@ sidebar_position: 1
 docker-compose up -d
 ```
 
-## Explore Appsmith (without docker-compose)
+:::info
+To complement automatic updates, Appsmith also has an auto-backup feature. Before an automatic update, Appsmith first creates a backup to ensure you can [roll back](/getting-started/setup/instance-management/appsmithctl#restore-appsmith-instance) an update to a previous version of Appsmith if necessary. For this auto-backup to work, please ensure that there is a minimum of 2 GB of free storage; otherwise, both the backup and update tasks fail.
+:::
 
-To quickly get Appsmith up and running, run the following command on your machine:
+This configuration runs an Appsmith instance and a Watchtower instance to keep Appsmith automatically up-to-date. Your installation is now configured to stay up-to-date automatically.
 
-### Community Edition
+---
 
-```bash
-docker run -d --name appsmith -p 80:80 -v "$PWD/stacks:/appsmith-stacks" --pull always appsmith/appsmith-ce
-```
+## Setup with Docker run
 
-### Business Edition
+To get Appsmith up and running, run the following command on your machine:
 
-```
-docker run -d --name appsmith -p 80:80 -v "$PWD/stacks:/appsmith-stacks" -e APPSMITH_LICENSE_KEY=<Add_YOUR_LICENSE_KEY> --pull always appsmith/appsmith-ee
-```
+### Community Edition setup
 
-This command will download the image and start Appsmith. Once the download is complete, the server should be up in under a minute. You can follow the logs with the following command:
+This command downloads the image and starts Appsmith.
 
 ```bash
-docker logs -f appsmith
+docker run -d --name appsmith -p 80:80 -v "$PWD/stacks:/appsmith-stacks" --pull always appsmith/appsmith-ce
 ```
 
-You should see a message `Appsmith is Running!` once the container is ready. The message is also logged and available in server logs(<mark>`stacks/logs/backend/backend.log`</mark>).
+#### Updating Appsmith
 
-![Appsmith is running message](/img/InstallationGuides__Docker__AppsmithRunningMessage.png)
+To update Appsmith manually, go to the root directory of your installation and run the following commands:
 
-:::tip
-If you are **unable** to **locate** the message but can **access** Appsmith via **localhost** or a **custom domain** you have **created**, then Appsmith is up and running.
-:::
-
-## Restarting Containers
-
-If your containers are failing to restart, you can execute the below script to bring them up:
+```
+docker rmi appsmith/appsmith-ce -f
+docker pull appsmith/appsmith-ce
+docker rm -f appsmith
+docker run -d --name appsmith -p 80:80 -v "$PWD/stacks:/appsmith-stacks" appsmith/appsmith-ce
+```
 
-<div class="downloadAssets">
-    <a target="_blank" download="restart-container.sh" href="/img/restart-container.sh">
-     <img src="/img/FileDownload.png" alt="Click to Download"/>
-    </a>
-</div>
-copy the script to your installation folder and make it executable
+Once the download is complete, the server should be up in under a minute. You can follow the logs with the following command:
```

Comment:
This same piece of information in repeated thrice in the document. I suggest we create a new section just above/below 'Restarting Containers' section named  'Docker Logs' and have this information about both logs and backend logs info mentioned under here